cloud/azure/bin/key-vault-ses

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env bash
```

Comment:
suggestion (blocking): a verb in the script name would be helpful, maybe `move-aws-ses-key-to-key-vault`

Reply:
oh wow i just realized the style guide calls for underscores instead of dashes in shell script file names. we have been doing the opposite. wups. definitely not something to address in this PR.